.github/workflows/test_integration.yml

```diff
@@ -0,0 +1,65 @@
+name: Integration Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+env: 
+  CGO_ENABLED: 0
+
+jobs:
+  check-cli-changes:
```

Comment:
This should save spinning up the `windows,linux,macOS` matrix if the CLI hadn't changed